.azure/gpu-tests-fabric.yml

```diff
@@ -101,6 +101,7 @@ jobs:
     - bash: |
         extra=$(python -c "print({'lightning': 'fabric-'}.get('$(PACKAGE_NAME)', ''))")
         pip install -e ".[${extra}dev]" pytest-timeout -U --find-links ${TORCH_URL}
+        pip install -U deepspeed
```

Comment:
TODO: remove

Reply:
This shall be covered in one of the extras, let me debug it...
```suggestion
```